hackathon_site/event/api_views.py

```diff
@@ -101,3 +106,53 @@ def post(self, request, *args, **kwargs):
         response_serializer = TeamSerializer(profile.team)
         response_data = response_serializer.data
         return Response(data=response_data, status=status.HTTP_200_OK,)
+
+
+class ProfileDetailView(mixins.UpdateModelMixin, generics.GenericAPIView):
+
+    queryset = Profile.objects.all()
+    serializer_class = ProfileSerializer
+    permission_classes = [FullDjangoModelPermissions]
+
+    @transaction.atomic
+    def patch(self, request, *args, **kwargs):
```

Comment:
This can be similar to the `CurrentProfileView`, in that it can use `self.partial_update`.

I don't think this one will require any special `update` method on the `ProfileSerializer`. In theory tech team shouldn't be modifying the `e_signature` or `acknowledge_rules` fields, but since we may eventually build the whole admin functionality into react we can keep it.